datafusion/functions/src/core/mod.rs

```diff
@@ -94,6 +94,7 @@ pub fn functions() -> Vec<Arc<ScalarUDF>> {
         nvl2(),
         arrow_typeof(),
         named_struct(),
+        get_field(),
```

Comment:
It might help to add the context of why this function is exported. Perhaps something like

```suggestion
       // Note most users invoke `get_field` indirectly via field access syntax 
       // like `my_struct_col['field_name']` results in a call to `get_field(my_struct_col, "field_name")`
       // It is also exposed directly for use cases such as serializing / deserializing plans with
       // the field access  desugared to calls to `get_field`
        get_field(),
```

Reply:
Added in  follow on PR: https://github.com/apache/datafusion/pull/11996